<?xml version="1.0"?>
<launch>
    <node pkg="waypoint_updater" type="waypoint_updater.py" name="waypoint_updater">
        <param name="nominal_deccel" value="1" />
        <param name="target_velocity" value="5.3" />
        <param name="stopping_distance_buffer" value="3" />
        <param name="lookahead_wps" value="200" /> 
    </node>
</launch>
